File: app/src/main/java/com/example/banbango_project/Model/ModelResultNearby.java
package com.example.banbango_project.Model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ModelResultNearby {
    @SerializedName("result")
    private List<ModelResults> modelResults;

    public List<ModelResults> getModelResults(){
        return modelResults;
    }

    public void setModelResults(List<ModelResults> modelResults) {
        this.modelResults = modelResults;
    }
}

File: app/src/main/java/com/example/banbango_project/Data/response/ModelResultNearby.kt
package com.example.banbango_project.data.model.response


import com.example.banbango_project.data.model.nearby.ModelResults
import com.google.gson.annotations.SerializedName

class ModelResultNearby {
    @SerializedName("results")
    lateinit var modelResults: List<ModelResults>
}